app/models/library_staff_record.rb

```diff
@@ -3,6 +3,7 @@
 # This class is responsible for storing and retrieving relevant
 # metadata from the library_staff_record table in the database
 class LibraryStaffRecord < ApplicationRecord
+  validates :puid, :netid, :name, :email, :title, :library_title, presence: true
```

Comment:
Validating that we have all the fields that have `NOT NULL` constraints, so that we can call `valid?` below